build-aux/com.obsproject.Studio.json

```diff
@@ -75,15 +75,15 @@
             "buildsystem": "cmake-ninja",
             "builddir": true,
             "config-opts": [
-                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-DOBS_CMAKE_VERSION=3",
```

Comment:
@PatTheMav does it matter if this is `3` or `3.0.0`?

Reply:
It should work nonetheless as long as CMake treats `3` identically to version `3.0.0`. I don't expect us to use any minor or patch versions anyway, so for the time being (and until legacy is removed) I'd be fine with it.

Reply:
Fine with me then.